app/src/main/java/org/liberty/android/fantastischmemo/ui/MiscTabFragment.java

```diff
@@ -362,6 +365,9 @@ public void onClick(DialogInterface arg0, int arg1){
         if(v == aboutButton){
             aboutUtil.createAboutDialog();
         }
+        if(v == favoriteButton){
+
+        }
```

Comment:
Why do you have this if statement?

Reply:
That is just there temporarily, it is where the backend logic will be placed